CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+  * 0.4.0:
+    - The official docker image is now on `smarkets/marge-bot` not (`smarketshq/marge-bot`).
+    - Add a --add-part-of option to tag commit messages with originating MR #48.
+    - Add a --git-timeout parameter (that takes time units); also add --ci-timeout
+      that deprecates --max-ci-time-in-minutes #58.
+    - Re-approve immediately after push #53.
+    - Always use --ssh-key-file if passed (never ssh-agent or keys from ~/.ssh) #61.
+    - Fix bad LOCALE problem in official image (hardcode utf-8 everywhere) #57.
+    - Don't blow up on logging bad jason responses #51.
```

Comment:
json perhaps instead of jason? :)

Reply:
@glensc Indeed -- thanks! I blame our CEO (I used to misspell him Json (Trost) all the time, but now the pendulum has swung too far the other way).